docs/geedocs/answer/3499938.html

```diff
@@ -62,18 +66,17 @@
 <li>SUSE Linux Enterprise Server 10</li>
 <li>Red Hat Enterprise Linux AS 4</li>
 </ul>
-<p>If you have previous patch releases of these operating systems, you should update to the latest patches. Google will continue to test the GEE software with the latest patch releases as they become available.</p>
-<p><strong>Software Platforms</strong>
-<p>These supported software platforms and versions have been tested and certified by the Google Earth Enterprise team:</p>
-<p>Version 5.0:</p>
+<p><strong>Software Platforms</strong></p>
+<p>The following software platforms are supported:</p>
+<p>Version 5.x:</p>
```

Comment:
Same comment here. Only open source versions past 5.3 are going to be supported.